demo/pom.xml

```diff
@@ -181,6 +181,7 @@
       <plugin>
         <groupId>org.sonatype.plugins</groupId>
```

Comment:
I think we don't need this staging maven plugin any more, as we don't deploy the artifacts into nexus public repo.

Reply:
done